packages/framework/presence/src/latestMapValueManager.ts

```diff
@@ -681,21 +694,17 @@ export function latestMap<
 	| InternalTypes.ManagerFactory<
 			RegistrationKey,
 			InternalTypes.MapValueState<T, Keys>,
-			LatestMapRaw<T, Keys>
+			LatestMap<T, Keys>
 	  >
 	| InternalTypes.ManagerFactory<
 			RegistrationKey,
 			InternalTypes.MapValueState<T, Keys>,
-			LatestMap<T, Keys>
+			LatestMapRaw<T, Keys>
```

Comment:
Reorder is not needed, but I prefer to order the preferred type, LatestMap, before the less-preferred type.